config/testgrids/config.yaml

```diff
@@ -77,6 +77,18 @@ test_groups:
 - name: test_pull_request_crio_e2e_rhel
   gcs_prefix: origin-federated-results/pr-logs/directory/test_pull_request_crio_e2e_rhel
 
+# dockershim
+- name: pull-kubernetes-node-e2e
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-kubernetes-node-e2e
+- name: pull-kubernetes-node-e2e-alpha
+  gcs_prefix: skip
```

Comment:
need to set to non-existing prefix for config/tests/testgrids/config_test.go:TestKubernetesProwInstanceJobsMustHaveMatchingTestgridEntries

Reply:
This is weird, we're removing them so why would we have to add these to testgrid configs?

Reply:
If you want I could take some time to document it in a comment...

Reply:
pretty sure the testgrid annotations on the prowjobs could be moved to the most recent branch that still has these jobs (most recent so this doesn't come back as old branches are removed from config)

this part:
```yaml
    annotations:
      testgrid-dashboards: sig-node-presubmits
      testgrid-tab-name: pr-node-kubelet-serial-kubetest2
```

Reply:
Sorry @BenTheElder I don't understand... test-infra needs to be always on the master branch, but the jobs corresponding to k/k 1.21 are defined in ‎config/jobs/kubernetes/sig-release/release-branch-jobs/1.21.yaml‎ for example, aren't they?
So I need to define the testgrid entries somewhere on test-infra master, no?